pkg/skaffold/survey/survey.go

```diff
@@ -17,26 +17,54 @@ limitations under the License.
 package survey
 
 import (
+	"context"
+	"fmt"
 	"io"
 	"os"
 
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/color"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
 )
 
 const (
-	Prompt = `Help improve Skaffold! Take a 10 seconds anonymous survey by running
+	Prompt = `Help improve Skaffold! Take a 10-second anonymous survey by running
    $skaffold survey`
```

Comment:
agree with @briandealwis, we shouldn't use `$` in strings. I think you can escape backticks like this:

```suggestion
	Prompt = `Help us improve Skaffold! Take an anonymous 10 second survey by running
   \`skaffold survey\``
```

Reply:
I removed the "$" sign.